service/src/main/java/com/commercetools/pspadapter/payone/transaction/paymentinadvance/BankTransferInAdvanceChargeTransactionExecutor.java

```diff
@@ -12,6 +12,19 @@
 
 import static io.sphere.sdk.payments.TransactionType.CHARGE;
 
+/**
+ * <b>Note:</b>: according to <i>TECHNICAL REFERENCE PAYONE Platform Channel Server API v2.92</i>
+ * (chapter <i>3.2.2 Initiating payment process (authorization)</i>)
+ * and <i>Sequenzdiagramme20170130</i> (chapter <i>Retailer – Zahlart Vorkasse mit Vorautorisierung</i>)
+ * this transaction type (Payone <i>authorization</i>) is <b>not</b> supported for this payment method:
+ * <pre>
+ *     Prepayment   Not supported by this request!
+ * </pre>
+ * <p>
+ * Looks like it is accepted by Payone service, but might be switch off in the future any time, also some logistic
+ * companies might have issues handling these transaction types for this method
+ * (see <a href="https://jira.commercetools.com/browse/CAR-2189">payment request for "Vorkasse" orders has wrong request type</a>)
```

Comment:
Please do not link internal resources here.

Reply:
4277ca61

Reply:
4277ca61